internal/config/config.go

```diff
@@ -110,7 +110,7 @@ func registerFlags() {
 		"info",
 		`The desired verbosity level for logging messages from nginx-agent. 
 		Available options, in order of severity from highest to lowest, are: 
-		panic, fatal, error, info, debug, and trace.`,
+		panic, fatal, error, info, debug and trace.`,
```

Comment:
Not a fan of the Oxford comma, eh?
https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/